src/components/Menu/GlobalSettings/SettingsModal.tsx

```diff
@@ -58,7 +58,7 @@ const SettingsModal: React.FC<InjectedModalProps> = ({ onDismiss }) => {
       title={t('Settings')}
       headerBackground="gradients.cardHeader"
       onDismiss={onDismiss}
-      style={{ maxWidth: '420px' }}
+      style={{ maxWidth: '420px', overflowY: 'scroll' }}
```

Comment:
Does this always overflow? If not, maybe `auto` is a better choice.

Reply:
Done